src/modules/admin/components/Organizations/OrganizationAddDomains.tsx

```diff
@@ -18,8 +18,8 @@ const MSG = defineMessages({
     id: 'admin.Organizations.OrganizationAddDomains.labelAddDomain',
     defaultMessage: 'Add New Domain',
   },
-  placeholderAddAdmins: {
-    id: 'admin.Organizations.OrganizationAddDomains.placeholderAddAdmins',
+  placeholderAddDomain: {
```

Comment:
Nice catch 👍 I think also we should probably move this component into the `Domains` dir, but can be left like this for now.